src/CommunityToolkit.Maui.Core/Essentials/FileFolderDialog.tizen.cs

```diff
@@ -0,0 +1,398 @@
+	}
+
+	public bool IsDirectory(string path)
+	{
+		if (((File.GetAttributes(path) & FileAttributes.Directory) == FileAttributes.Directory))
+		{
+			return true;
+		}
+		return false;
+	}
+
+	protected override View CreateContent()
+	{
+		Layout = new LinearLayout
+		{
+			VerticalAlignment = VerticalAlignment.Center,
+			HorizontalAlignment = HorizontalAlignment.Center
+		};
+		BackgroundColor = new TColor(0.1f, 0.1f, 0.1f, 0.5f).ToNative();
+		
+		var isHorizontal = Window.Instance.WindowSize.Width > Window.Instance.WindowSize.Height;
+		var margin1 = (ushort)20d.ToPixel();
+		var margin2 = (ushort)10d.ToPixel();
+		var radius = 8d.ToPixel();
+
+		var content = new View
+		{
+			CornerRadius = radius,
+			BoxShadow = new Shadow(20d.ToPixel(), TColor.Black.ToNative()),
+			Layout = new LinearLayout
+			{
+				VerticalAlignment = VerticalAlignment.Center,
+				HorizontalAlignment = HorizontalAlignment.Center,
+				LinearOrientation = LinearLayout.Orientation.Vertical,
+			},
+			SizeWidth = Window.Instance.WindowSize.Width * (isHorizontal ? 0.5f : 0.8f),
+			BackgroundColor = TColor.White.ToNative(),
+		};
+
+		content.Add(new Label
+		{
+			Text = isFileSelectionMode ? fileSaveString : selectFolderString,
+			Margin = new Extents(margin1, margin1, margin1, margin2),
+			WidthSpecification = LayoutParamPolicies.MatchParent,
+			HorizontalTextAlignment = Tizen.UIExtensions.Common.TextAlignment.Start,
+			VerticalTextAlignment = Tizen.UIExtensions.Common.TextAlignment.Center,
+			FontAttributes = Tizen.UIExtensions.Common.FontAttributes.Bold,
+			TextColor = TColor.Black,
+			PixelSize = 21d.ToPixel(),
+		});
+
+		content.Add(new View
+		{
+			BackgroundColor = TColor.FromHex("#cccccc").ToNative(),
+			SizeHeight = 1.5d.ToPixel(),
+			WidthSpecification = LayoutParamPolicies.MatchParent,
+		});
+
+		directoryScrollView = new ScrollView
+		{
+			Margin = new Extents(margin1, margin1, 0, 0),
+			VerticalScrollBarVisibility = Tizen.UIExtensions.Common.ScrollBarVisibility.Default,
+			WidthSpecification = LayoutParamPolicies.MatchParent,
+		};
+		directoryScrollView.ContentContainer.Layout = new LinearLayout
+		{
+			LinearOrientation = LinearLayout.Orientation.Vertical,
+		};
+		content.Add(directoryScrollView);
+
+		UpdateDirectoryScrollView(initialFileFolderPath);
+
+		content.Add(new View
+		{
+			BackgroundColor = TColor.FromHex("#cccccc").ToNative(),
+			SizeHeight = 1.5d.ToPixel(),
+			WidthSpecification = LayoutParamPolicies.MatchParent,
+		});
+
+		if (isFileSelectionMode)
+		{
+			var newFolderButton = new Button
+			{
+				Focusable = true,
+				Text = "New Folder",
+				TextColor = TColor.Black,
+				BackgroundColor = TColor.Transparent.ToNative(),
+				Margin = new Extents(margin1, margin1, 0, 0),
+				WidthSpecification = LayoutParamPolicies.MatchParent
+			};
+			newFolderButton.Clicked += async (sender, args) =>
+			{
+				try
+				{
+					var newDirectoryName = await new PromptPopup("New Folder Name", "").Open();
```

Comment:
Is this some title? If so isn't localized and will not be aligned with other platforms that have the dialog out of the box, like Windows